src/makeCalculator.js

```diff
@@ -4,7 +4,28 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+
+    add: (number) => result + number,
+    subtract: (number) => result - number,
+    multiply: (number) => result * number,
+    divide: (number) => result / number,
```

Comment:
Use the assignment operators with the arithmetic operation to update the `result`. According to the checklist [CODE STYLE], you should use `/=` instead of `/` for dividing the existing value.